pkg/apis/application/v1alpha1/generated.proto

```diff
@@ -644,6 +644,8 @@ message ResourceDiff {
   optional string diff = 7;
 
   optional bool hook = 8;
+
+  optional string expectedLiveState = 9;
```

Comment:
What do you think about `predictedLiveState` instead of `expectedLiveState` ?

Reply:
Or even `predictedState` ?

Reply:
I agree. It is easy to confuse expected and target state. So the predicted state is better.

Reply:
I've tried to fix the failed e2e test and realized we need one more field that stores normalized live state. Diff massages live object before comparing it so we have to send both normalized live state as well as predicted. 

Want to get your opinion about this change.